### ISSUE TITLE
Is it possible to change the Twitter fixed link from fxtwitter to fixupx?

### ISSUE THREAD

**Issue by gamania31610** [Kyrela/FixTweetBot#16]
Dear developer

I've noticed that some links do not display properly on fxtwitter, but they work when I change the URL to fixupx. I'm wondering if it might be because the fixtwitter API or URL is outdated.

Fixupx is also provided by FixTweet/FxTwitter, so I would appreciate it if you could confirm whether this is feasible.

![49498](https://github.com/user-attachments/assets/884dae2f-d920-4192-9851-8fc1c1e18927)

**Comment by Kyrela** [comment]
Hello! I think it was apparently an isolated problem, because I can't reproduce it. Do you have any other examples?

**Comment by gamania31610** [comment]
I just tested the link again, and it’s displaying properly now. This afternoon, I tried many times, but it wouldn’t display, which is really strange. I had the same issue early yesterday morning with this [link](https://x.com/amsrntk3/status/1871896971356913895), but when I reposted it just now, it worked fine. I’m guessing, could it be that some posts on fxtwitter take longer to process compared to fixupx?

![444](https://github.com/user-attachments/assets/065bbebc-d52d-42a8-9884-555bc60dba79)

**Comment by gamania31610** [comment]
I just tested the link again, and it’s displaying properly now. This afternoon, I tried many times, but it wouldn’t display, which is really strange. I had the same issue early yesterday morning with this [link](https://x.com/amsrntk3/status/1871896971356913895), but when I reposted it just now, it worked fine. I’m guessing, could it be that some posts on fxtwitter take longer to process compared to fixupx?

![444](https://github.com/user-attachments/assets/aed2f65f-d627-47fb-8e95-674c79158bb1)
